tensorflow/contrib/cmake/README.md

```diff
@@ -13,12 +13,13 @@ Linux.
 Current Status
 --------------
 
-CMake can be used to build TensorFlow on Windows. See the [getting started documentation](https://www.tensorflow.org/install/install_windows)
-for instructions on how to install a pre-built TensorFlow package on Windows.
+CMake can be used to build TensorFlow on all platforms. See the [getting started documentation](https://www.tensorflow.org/install/install_windows)
+for instructions on how to install a pre-built TensorFlow package on Windows and Linux. Procedure in MacOS is similar to the Linux build.
 
 ### Current known limitations
 * It is not possible to load a custom Op library.
 * GCS file system is not supported.
+* Debug build is not available since python no longer release debug library.
```

Comment:
Replace "since python no longer release debug library." with "since Python for Windows is no longer distributed with a debug library."